<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="cam_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>cam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_up_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>90</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_down_button">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_right_button">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>160</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_left_button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>170</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flame value1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Gas value1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gas_value1_label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>380</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="flame_value1_label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>440</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deactivate_button">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>680</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>DEACTIVATE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gas_led_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="flame_led_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>470</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="log_tableWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>560</y>
     <width>401</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>540</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DB state</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ventilation_button">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>680</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Ventilation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>766</x>
     <y>360</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gas value2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="flame_value2_label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>440</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>420</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Flame value2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gas_value2_label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>380</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_camera">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gas1_test">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>gas1_test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gas2_test">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>gas2_test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flame1_test">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>flame1_test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="flame2_test">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>flame2_test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateValue">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>updatedValue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end_program_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>endProgram</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
